WorkloadManagementSystem/Client/JobManagerClient.py

```diff
@@ -0,0 +1,24 @@
+""" Class that contains client access to the JobManager handler. """
+
+from DIRAC.Core.Base.Client import Client
+
+
+class JobManagerClient(Client):
+  """JobManagerClient sets url for the JobManagerHandler.
+  """
+
+  def __init__(self, url=None, **kwargs):
+    """
+    Sets URL for JobManager handler
+    :param self: self reference
```

Comment:
Before this line you need a line break.

Reply:
I backported this file from the integration branch as-is. To avoid possible merging issues I would prefer to leave it as it is.